File: Chp3/app/src/main/java/scmDev/com/MainActivity.java
package scmDev.com;

import android.os.Bundle;
import android.view.View;
import android.widget.Toast;

import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

    int clickCount = 0;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        Toast.makeText(getApplicationContext(), "프로그래밍을 시작해보자!", Toast.LENGTH_LONG).show();

        findViewById(R.id.button).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                clickCount += 1;

                Toast.makeText(getApplicationContext(),"Click Count: " + clickCount, Toast.LENGTH_SHORT).show();
            }
        });
    }
}

File: Lecture1/Example1/app/src/main/java/com/example/textviewkt/MainActivity.kt
package com.example.textviewkt

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {



    override fun onCreate(savedInstanceState: Bundle?) { // 앱이 최초 실행되었을 때 실행
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main) // XML 화면 뷰를 연결한다.

        tv_title.setText("Hi Android...!")

    }
}